fdbserver/workloads/TenantManagementWorkload.actor.cpp

```diff
@@ -881,17 +903,16 @@ struct TenantManagementWorkload : TestWorkload {
 						}
 					}
 
-					if (!tenants.empty()) {
+					if (!tenants.empty() && operationType == OperationType::METACLUSTER) {
```

Comment:
I think this also needs to check for all operation types, but for non-metacluster cases this can just observe the absence of the tenant.